src/index-pipelines.js

```diff
@@ -32,21 +32,19 @@ async function run(params, path) {
   const action = `${pkg}/${type}_json${version ? `@${version}` : ''}`;
 
   log.info(`Invoking ${action} for path: ${path}`);
-  const { activationId, response: { result } } = await openwhisk().actions.invoke({
+  const { activationId, response: { result } } = await ow().actions.invoke({
     name: `${action}`,
     blocking: true,
     params: {
       owner, repo, ref, path,
     },
   });
 
-  if (!result.body.docs) {
-    const message = `${action} (activation id: ${activationId}) returned no documents`;
+  if (!result.body) {
+    const message = `${action} (activation id: ${activationId}) returned no body`;
```

Comment:
`index-pipelines` returns a body consisting of one entry per index definition. That entry then contains a `docs` item, e.g.:
```
  "blog-posts": {
    "docs": [{
      "fragmentID": "",
      "author": "James Fox",
      ...
```